source/ems_prims.cpp

```diff
@@ -514,7 +514,16 @@ double Zone::getVectAngle(std::complex<double> U, std::complex<double> V)
 {
     double dot_prod = U.real() * V.real() + U.imag() * V.imag();
     double mag_prod = abs(U) * abs(V);
-    double angle = acos(dot_prod / mag_prod);
+    double angle = 0;
+
+    if ((dot_prod / mag_prod)<=-1.0) {
```

Comment:
Please adjust spacing and remove redundant parentheses so that: `if (dot_prod / mag_prod <= -1.0) {`